src/lib/logstore/log_store_family.cpp

```diff
@@ -286,8 +294,8 @@ nlohmann::json LogStoreFamily::dump_log_store(const log_dump_req& dump_req) {
     if (dump_req.log_store == nullptr) {
         folly::SharedMutexWritePriority::ReadHolder holder(m_store_map_mtx);
```

Comment:
nitpick: now that `mutable` is applied, probably we can make this dump_log_store as const function as well?
you can ignore this comment if it becomes messy or you want to keep as it is.